tasks/validate/validate.yml

```diff
@@ -16,8 +16,15 @@
     fail_msg: The value {{ nms_setup }} you used for `nms_setup` is not allowed. Try one of {{ nms_setup_vars | join(', ') }}.
   tags: nms_check_support
 
+- name: Check if 'nms_service_state' is an allowed value
+  ansible.builtin.assert:
+    that: nms_service_state in nms_service_state_vals
```

Comment:
Maybe we can use `nms_service_state_vars` and get rid of nms_service_state_vals`?

Reply:
Good call, this has been updated.